.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Overview
+_A brief description of the pull request. Try to keep it non-technical. Please mark your pull request with the appropriate label(s)_
```

Comment:
@mickadoo yeah, it can be link to some other page. My concern is that "appropriate labels" is too vague and might have different meanings for different people